src/main/java/com/exasol/adapter/dynamodb/mapping/TableMappingDefinition.java

```diff
@@ -11,16 +12,28 @@
  * table in the Exasol Virtual Schema. Typically it also represents a DynamoDB table. But it can also represent the data
  * from a nested list or object. See {@link #isRootTable()} for details.
  */
-public class TableMappingDefinition {
+public class TableMappingDefinition implements Serializable {
+    private static final long serialVersionUID = 3568807256753213582L;
     private final String exasolName;
-    private final List<AbstractColumnMappingDefinition> columns;
+    private final String remoteName;
+    private final transient List<AbstractColumnMappingDefinition> columns; // The columns are serialized separately in
```

Comment:
Then it's strange that the list belongs here but is being serialized somewhere else.

Reply:
well, there are two tree structures that have the analog structure but different data. Serializing each node into the coresponding one is helpful as Exasol gives me the right nodes automatically.